src/fsharp/FSharp.Core/control.fs

```diff
@@ -729,7 +729,7 @@ namespace Microsoft.FSharp.Control
                     )
                     
         let startThreadWithTrampoline (trampolineHolder:TrampolineHolder) (f : unit -> FakeUnitValue) =
-#if FX_NO_THREADPOOL
+#if FX_THREADPOOL
                 if not (ThreadPool.QueueUserWorkItem((waitCallbackForQueueWorkItemWithTrampoline trampolineHolder), f |> box)) then
```

Comment:
`FX_NO_THREADPOOL` is actually accurate.  The `ThreadPool` used here is a "fake" version that is defined only for use when the framework doesn't actually contain `ThreadPool` (defined [here](https://github.com/Microsoft/visualfsharp/blob/fsharp4/src/fsharp/FSharp.Core/control.fs#L480-L483)).